src/i18n/utils.ts

```diff
@@ -32,5 +32,10 @@ export function t(key: I18nKey, ...args: any[]) {
  * @param args The arguments to format the string with.
  */
 export function tA11y(key: I18nKey, ...args: any[]) {
-  return striptags(t(key, ...args));
+  return striptags(
+    t(key, ...args)
+      .replace(/\.{3}/g, 'to')
+      .replace(/<kbd>(\S+)<\/kbd>/g, `'$1'`)
+      .replace(/'\?'/g, `'QUESTION MARK'`),
```

Comment:
Is it a problem that these are always english?

Reply:
oh I think it should be in i18n indeed.